src/docs/user/references/Variables.adoc

```diff
@@ -162,22 +160,37 @@ Examples:
+
 --
-* *PA:JSON* : variable syntax must be a valid JSON expression as defined in http://www.json.org/[JSON doc]. +
-Examples: PA:JSON will accept {"name": "John", "city":"New York"} and empty values like {} or [{},{}], but not ["test" : 123] (Unexpected character ':') and {test : 123} (Unexpected character 't').
-* *PA:NOT_EMPTY_STRING* : variable must be provided with a non-empty string value.
-* *PA:HIDDEN* : variable which allows the user to securely enter its value (i.e., each character is shown as an asterisk, so that it cannot be read.) while submitting the workflow.
-* *PA:CREDENTIAL* : variable whose value is a key of the <<_third_party_credentials, ProActive Scheduler Third-Party Credentials>> (which are stored on the server side in encrypted form). The variable allows the user to access a credential through `credentials.get(variables.get("MY_CRED_KEY"))` instead of in plain-text in the workflow definition. At workflow submission, the scheduler server verifies that the key exists in the 3rd party credentials of the user.
-* *PA:GLOBAL_FILE* : variable whose value is the relative path of a file in the <<_global_and_user_spaces, Global Data Space>>. At workflow submission, the scheduler server verifies that the file exists in the global dataspace.
-* *PA:USER_FILE* : variable whose value is the relative path of a file in the <<_global_and_user_spaces, User Data Space>>. At workflow submission, the scheduler server verifies that the file exists in the user dataspace.
-* *PA:GLOBAL_FOLDER* : variable whose value is the relative path of a folder in the <<_global_and_user_spaces, Global Data Space>>. At workflow submission, the scheduler server verifies that the folder exists in the global dataspace. Note, the variable value should not end with a slash to avoid the problem of duplicate slash in its usage.
-* *PA:USER_FOLDER* : variable whose value is the relative path of a folder in the <<_global_and_user_spaces, User Data Space>>. At workflow submission, the scheduler server verifies that the folder exists in the user dataspace. Note, the variable value should not end with a slash to avoid the problem of duplicate slash in its usage.
+[none]
+* {empty}
 
+** *PA:CREDENTIAL* : variable whose value is a key of the <<_third_party_credentials, ProActive Scheduler Third-Party Credentials>> (which are stored on the server side in encrypted form). The variable allows the user to access a credential through `credentials.get(variables.get("MY_CRED_KEY"))` instead of in plain-text in the workflow definition. At workflow submission, the scheduler server verifies that the key exists in the 3rd party credentials of the user.
+** *PA:CRON* : variable syntax must be a valid cron expression as defined in the  http://www.sauronsoftware.it/projects/cron4j/manual.php#p02[cron4j manual]. +
+Examples: PA:CRON will accept "5 * * * \*" but not "* * * *" (missing minutes sub-pattern).
```

Comment:
Examples: PA:CRON will accept "5 * * * *" but not "* * * *" (missing minutes sub-pattern).
This model can be used for example to control the provided value to the `loop` control flow parameter.